security/src/main/scala/pl/touk/nussknacker/ui/security/oidc/OidcAuthenticationResources.scala

```diff
@@ -12,10 +12,9 @@ class OidcAuthenticationResources(name: String, realm: String, configuration: Oi
       name = name,
       realm = realm,
       service = new CachingOAuth2Service(
-        new UserMappingOAuth2Service(
+        new UserMappingOAuth2Service[OpenIdConnectUserInfo, DefaultOidcAuthorizationData](
           new OidcService(configuration),
-          (userInfo: OpenIdConnectUserInfo) =>
-            OpenIdConnectProfile.getAuthenticatedUser(userInfo, configuration.oAuth2Configuration)
+          new OpenIdConnectProfileAuthentication(configuration.oAuth2Configuration)
```

Comment:
shouldn't we use the `Oidc` abbreviation in the case of this class too? To be consistent. So, maybe `OidcProfileAuthentication`?

Reply:
Yeah, I don't know what concept someone has about having two prefixes for oidc. I used `Oidc` abbreviation in all places where `OpenIdConnect` was and also moved all `oidc*` classes to `oidc` package